includes/class-fs-api.php

```diff
@@ -617,14 +592,52 @@ static function is_api_error( $result ) {
          * @since  2.0.0
          *
          * @param mixed $result
+         * @param bool  $ignore_message
          *
          * @return bool Is API result contains an error.
          */
-        static function is_api_error_object( $result ) {
+        static function is_api_error_object( $result, $ignore_message = false ) {
             return (
                 is_object( $result ) &&
                 isset( $result->error ) &&
-                isset( $result->error->message )
+                ( $ignore_message || isset( $result->error->message ) )
+            );
+        }
+
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since 2.5.4
+         *
+         * @param mixed $response
+         *
+         * @return bool
+         */
+        static function is_ssl_error_response( $response ) {
+            $http_error = null;
+
+            if ( $response instanceof WP_Error ) {
+                if (
+                    isset( $response->errors ) &&
+                    isset( $response->errors['http_request_failed'] )

```

Comment:
If I'm not mistaken, you previously mentioned that the WP conventions also include adding space within square brackets, i.e., `$response->errors[ 'http_request_failed' ]`. If true, also applies to the statement within the `if`.

Reply:
@vovafeldman Spaces are used only when the index is a variable:
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#space-usage